pkg/cmd/sync/tasks.go

```diff
@@ -0,0 +1,59 @@
+package sync
+
+import (
+	"context"
+	"fmt"
+
+	msg "github.com/aziontech/azion-cli/messages/sync"
+	api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"go.uber.org/zap"
+)
+
+var (
+	C    context.Context
+	Opts *contracts.ListOptions
+)
+
+func (synch *SyncCmd) SyncResources(f *cmdutil.Factory, info contracts.SyncOpts) error {
+	C = context.Background()
+	Opts = &contracts.ListOptions{
+		PageSize: 1000,
+		Page:     1,
+	}
+	err := synch.syncRules(info, f)
+	if err != nil {
+		return fmt.Errorf(msg.ERRORSYNC, err.Error())
+	}
+	return nil
+}
+
+func (synch *SyncCmd) syncRules(info contracts.SyncOpts, f *cmdutil.Factory) error {
+
+	client := api.NewClient(f.HttpClient, f.Config.GetString("api_url"), f.Config.GetString("token"))
+	resp, err := client.ListRulesEngine(C, Opts, info.Conf.Application.ID, "request")
+	if err != nil {
+		return err
+	}
+
+	for _, rule := range resp.Results {
+		if id := info.RuleIds[rule.Name]; id > 0 || rule.Name == "Default Rule" {
+			// if remote rule is also on local environment, no action is needed
```

Comment:
Considering we're only checking the ID... What if something in the rule was changed, like the behavior for example? 🤔

Reply:
This command is used for another purpose:
- We check if there are any rules that were CREATED remotely, but not found on your local azion.json file. It does not consider any change in behavior in existing rules

Reply:
![ata](https://imageproxy.ifunny.co/crop:x-20,resize:640x,quality:90x75/images/f44afc2791468c3a4daee9605de2ed3d1990dcfd018106f48a440c5858fd9269_1.jpg)